src/lang/en-US.ts

```diff
@@ -32,13 +33,17 @@ const CommonMessages: ResolvedIntlConfig['messages'] = {
 
     // Data
     'data.name': `Name`,
+    'data.firstName': `First name`,
+    'data.lastName': `Last name`,
     'data.description': `Description`,
     'data.status': `Status`,
     'data.type': `Type`,
     'data.maintainer': `Maintainer`,
     'data.updated_at': `Updated`,
     'data.email': `Email`,
     'data.display_name': `User Name`,
+    'data.company': `Company`,
+    'data.register.intendedUse': `Describe your use case`,
```

Comment:
The original intention of the `data` chunk was actually for labels/headers when outputting details in the UI... no so much for when putting labels on forms. 

This is also a place where I'd probably not add the concept of `register` and just move the `intendedUse` to the register chunk below.

Reply:
Ah, good to know. The classification of `Data` was a tad vague and I misinterpreted the use. Will update.